Automattic/jetpack
# Social: Add Social Share Status modal for published posts

Fixes https://github.com/Automattic/jetpack-reach/issues/532

## Proposed changes:

* Added new components for the modal
* Ordered the shares so latest comes first

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/532
## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Add a new post and share it to some platforms
* In the editor click Sharing status.
* In the modal you should see the shares and if they failed you should see a button to retry, and an error in the tooltip
* Reshare the post to some other network
* Without refreshing the page run `await wp.data.dispatch( 'jetpack-social-plugin').invalidateResolution( 'getPostShareStatus', [<POST_ID>] );` which invalidates the store and retrives the shares again
* The new share should show up in the logs. You might have to wait a bit before issuing this command
